UABEAvalonia/Forms/PluginWindow.axaml.cs

```diff
@@ -34,7 +34,7 @@ public PluginWindow(Window win, AssetWorkspace workspace, List<AssetContainer> s
             this.selection = selection;
 
             plugInfs = plugLoader.GetPluginsThatSupport(workspace.am, selection);
-            boxPluginList.Items = plugInfs;
+            boxPluginList.ItemsSource = plugInfs;
```

Comment:
We should use `ItemsSource` here, but Items is already assigned in the xaml file. Just replace the ListBox for `boxPluginList` to this: `<ListBox Margin="10,10,10,45" Name="boxPluginList" />`